yarn-deployment/yarn_dbt.py

```diff
@@ -170,6 +174,23 @@ def generate_yarn_shell_command(app_name):
         app_name
     )
 
+    # Set environment variable for dbt deployment
+    set_environment_variables_start = "echo -n '{}: Setting env blob for deployment start: '; date +'%Y-%m-%d:%H:%M:%S'".format(
+        app_name
+    )
+    DBT_DEPLOYMENT_ENV = "env:{},version:{},description:{}".format(
+        "yarn",
+        "1.2.0",
```

Comment:
is this the yarn version or the yarn_dbt version ?

Reply:
This is the package version. Yarn version  I haven't added to the blob yet.